src/main/java/net/snowflake/client/config/SFClientConfigParser.java

```diff
@@ -34,21 +35,26 @@ public static SFClientConfig loadSFClientConfig(String configFilePath) throws IO
     String derivedConfigFilePath = null;
     if (configFilePath != null && !configFilePath.isEmpty()) {
       // 1. Try to read the file at  configFilePath.
+      logger.info("Using config file specified from connection string: {}", configFilePath);
       derivedConfigFilePath = configFilePath;
     } else if (System.getenv().containsKey(SF_CLIENT_CONFIG_ENV_NAME)) {
       // 2. If SF_CLIENT_CONFIG_ENV_NAME is set, read from env.
-      derivedConfigFilePath = systemGetEnv(SF_CLIENT_CONFIG_ENV_NAME);
+      String filePath = systemGetEnv(SF_CLIENT_CONFIG_ENV_NAME);
+      logger.info("Using config file specified from environment variable: {}", filePath);
```

Comment:
can we change `config file` log parts to `configuration file`?

Reply:
In the other drivers we use the message in such format. This would keep the parity between them.